--- isolinux.cfg	2022-07-21 16:17:19.789322965 +0300
+++ isolinux-p.cfg	2022-07-21 17:05:32.069294947 +0300
@@ -69,8 +69,13 @@
   kernel vmlinuz
   append initrd=initrd.img inst.stage2=hd:LABEL=redos-MUROM-7.3.1\x20x86_64 inst.multilib quiet inst.kdump_addon=off
 
+label linux
+  menu label ^2. Kickstart RED OS
+  kernel vmlinuz
+  append initrd=initrd.img inst.stage2=hd:LABEL=redos-MUROM-7.3.1\x20x86_64 quiet inst.kdump_addon=off inst.ks=hd:LABEL=redos-MUROM-7.3.1\x20x86_64:/ks.cfg
+
 label check
-  menu label ^2. Проверить установочный носитель и установить RED OS
+  menu label ^3. Проверить установочный носитель и установить RED OS
   menu default
   kernel vmlinuz
   append initrd=initrd.img inst.stage2=hd:LABEL=redos-MUROM-7.3.1\x20x86_64 inst.multilib rd.live.check quiet
@@ -125,4 +130,4 @@
 
 label local
   menu label ^0. Загрузка с локального диска
-  localboot 0xffff
\ No newline at end of file
+  localboot 0xffff
